packages/lambdapi/lambdapi.2.2.1/opam

```diff
@@ -54,7 +54,7 @@ depends: [
   "dedukti" {with-test & >= "2.7"}
   "bindlib" {>= "5.0.1"}
```

Comment:
```suggestion
  "bindlib" {>= "6.0.0"}
```
Due to
```

#=== ERROR while compiling lambdapi.2.2.1 =====================================#
# context              2.2.0~alpha3~dev | linux/x86_64 | ocaml-base-compiler.4.08.1 | pinned(https://github.com/Deducteam/lambdapi/releases/download/2.2.1/lambdapi-2.2.1.tbz)
# path                 ~/.opam/4.08/.opam-switch/build/lambdapi.2.2.1
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p lambdapi -j 71 @install
# exit-code            1
# env-file             ~/.opam/log/lambdapi-7-7443ee.env
# output-file          ~/.opam/log/lambdapi-7-7443ee.out
### output ###
# File "/home/opam/.opam/4.08/lib/biniou/biniou.dune", line 1, characters 0-0:
# Warning: .dune files are ignored since 2.0. Reinstall the library with dune
# >= 2.0 to get rid of this warning and enable support for the subsystem this
# library provides.
# File "/home/opam/.opam/4.08/lib/ppx_derivers/ppx_derivers.dune", line 1, characters 0-0:
# Warning: .dune files are ignored since 2.0. Reinstall the library with dune
# >= 2.0 to get rid of this warning and enable support for the subsystem this
# library provides.
#       ocamlc src/handle/.handle.objs/byte/handle__Proof.{cmi,cmo,cmt} (exit 2)
# (cd _build/default && /home/opam/.opam/4.08/bin/ocamlc.opt -w -40 -g -bin-annot -I src/handle/.handle.objs/byte -I /home/opam/.opam/4.08/lib/bindlib -I /home/opam/.opam/4.08/lib/camlp-streams -I /home/opam/.opam/4.08/lib/gen -I /home/opam/.opam/4.08/lib/menhirLib -I /home/opam/.opam/4.08/lib/num -I /home/opam/.opam/4.08/lib/ocaml-migrate-parsetree -I /home/opam/.opam/4.08/lib/ocaml/compiler-libs -I /home/opam/.opam/4.08/lib/ppx_derivers -I /home/opam/.opam/4.08/lib/ppx_tools_versioned -I /home/opam/.opam/4.08/lib/ppx_tools_versioned/metaquot_408 -I /home/opam/.opam/4.08/lib/pratter -I /home/opam/.opam/4.08/lib/result -I /home/opam/.opam/4.08/lib/sedlex -I /home/opam/.opam/4.08/lib/sedlex/ppx -I /home/opam/.opam/4.08/lib/stdlib-shims -I /home/opam/.opam/4.08/lib/timed -I /home/opam/.opam/4.08/lib/uchar -I /home/opam/.opam/4.08/lib/why3 -I src/common/.common.objs/byte -I src/common/.common.objs/native -I src/core/.core.objs/byte -I src/core/.core.objs/native -I src/lplib/.lplib.objs/byte -I src/lplib/.lplib.objs/native -I src/parsing/.parsing.objs/byte -I src/parsing/.parsing.objs/native -I src/tool/.tool.objs/byte -I src/tool/.tool.objs/native -no-alias-deps -open Handle -o src/handle/.handle.objs/byte/handle__Proof.cmo -c -impl src/handle/proof.ml)
# File "src/handle/proof.ml", line 58, characters 29-49:
# 58 |       let add_name c (n,_) = Bindlib.reserve_name n c in
#                                   ^^^^^^^^^^^^^^^^^^^^
# Error: Unbound value Bindlib.reserve_name
```

Reply:
@fblanqui I am not allowed to make changes to this PR, can you commit this suggestion? Then I can merge